tests/functional/u/unsupported/unsupported_version_for_final.py

```diff
@@ -0,0 +1,11 @@
+"""Tests for the use of typing.final whenever the py-version is set < 3.8"""
+# pylint: disable=missing-class-docstring, too-few-public-methods, missing-function-docstring, no-name-in-module
+
+from typing import final
+
+
+@final # [using-final-in-unsupported-version]
+class MyClass:
+    @final # [using-final-in-unsupported-version]
+    def my_method(self):
```

Comment:
What happens if someone import `typing` and thus uses `@typing.final` instead?

Reply:
Thank you for the review 👍 . I'll resolve the other conversations (let me know if any concerns) but leave this one open and it will be the next focus to make the logic more robust for checking if `typing.final` is the intended use of the decorator which is being used for py-version < 3.8 on a Python version < 3.8.